pkgs/by-name/no/nordvpn/package.nix

```diff
@@ -0,0 +1,112 @@
+{
+  pkgs,
+  system ? stdenv.hostPlatform.system,
+  autoPatchelfHook,
+  dpkg,
+  fetchurl,
+  lib,
+  stdenv,
+  sysctl,
+  iptables,
+  iproute2,
+  libxml2,
+}:
+
+let
+  pname = "nordvpn";
+  version = "3.20.2";
+
+  archMap = {
+    "x86_64-linux" = "amd64";
+    "i686-linux" = "i386";
+    "aarch64-linux" = "arm64";
+    "armv7l-linux" = "armhf";
+    "armv5tel-linux" = "armel";
+  };
+
+  baseURL = "https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/n/nordvpn";
+  systemArch = archMap.${system} or (throw "${system} is unsupported");
+
+  thisPkg = stdenv.mkDerivation {
+    inherit pname version;
+    src = fetchurl {
+      url = "${baseURL}/${pname}_${version}_${systemArch}.deb";
+      hash = "sha256-2eLVr0aQpMhIMCrkyIRcCabXc197GJZ31Vl5BND7Di8=";
+    };
+
+    nativeBuildInputs = [
+      dpkg
+      autoPatchelfHook
+    ];
+    buildInputs =
+      [ libxml2 ]
+      ++ (with pkgs; [
```

Comment:
dependencies should be passed via the function definition at the top of the file, not with `pkgs.`